.travis.yml

```diff
@@ -1,33 +1,61 @@
 language: php
 
-sudo: false
```

Comment:
this line is no longer necessary, it was useful during the upgrade of their containers but has absolutely no effect (unless you specify `true`)